.github/workflows/e2e-run.yml

```diff
@@ -134,7 +134,7 @@ jobs:
   reports:
     runs-on: ubuntu-latest
     needs: tests
-    if: ${{ !cancelled() && github.repository == 'kafbat/kafka-ui' }}
+    if: ${{ !cancelled() && github.repository == 'kafbat/kafka-ui' && github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
do we even need `github.repository == 'kafbat/kafka-ui'` anymore?

Reply:
oh ya we don't need it anymore, I will fix this and push the changes